tests/assemblers/test_lightgbm.py

```diff
@@ -12,6 +12,12 @@ def test_binary_classification():
     assembler = assemblers.LightGBMModelAssembler(estimator)
     actual = assembler.assemble()
 
+    estimator.booster_.save_model('lgb_model.txt')
```

Comment:
Where this file is saved? Should we use temporary directory instead? Consider using the `tmp_dir()` context manager from the `tests.utils` package. Like:
```
import os
from tests import utils

with utils.tmp_dir() as tmp_dirpath:
  lab_model_path = os.path.join(tmp_dirpath, 'lgb_model.txt')
  ...
```